File: ios/reactnativebasic/module/CalendarManager.m
//
//  CalendarManage.m
//  reactnativebasic
//
//  Created by SMG on 2023/04/26.
//

#import <Foundation/Foundation.h>
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(CalendarManager, NSObject)

RCT_EXTERN_METHOD(addEvent:(NSString *)name location:(NSString *)location date:(nonnull NSNumber *)date)

@end

File: ios/reactnativebasic/module/CalendarManager.swift
//
//  CalendarManager.swift
//  reactnativebasic
//
//  Created by SMG on 2023/04/26.
//

import Foundation

@objc(CalendarManager)
class CalendarManager: NSObject {

  @objc(addEvent:location:date:)
  func addEvent(name: String, location: String, date: NSNumber) -> Void {
    // Date is ready to use!
    print(name, location, date)
   }
}
